lib/applications/model.js

```diff
@@ -0,0 +1,89 @@
+/**
+ *  @fileoverview Applications model
+ *
+ *  @author  Scott Nath
+ *
+ */
+'use strict';
+
+const types = require('punchcard-content-types');
+const config = require('config');
+
+/**
+ * Applications data entry structure
+ *
+ * @returns {object} - content structure to match requirements of punchcard-content-types
+ */
+const structure = {
+  name: 'Applications',
+  description: 'Contains webhook applications',
+  id: 'applications',
+  identifier: 'name',
+  attributes: [
+    {
+      type: 'text',
+      id: 'name',
+      name: 'Name',
+      required: 'save',
+      description: 'Application name',
+      inputs: {
+        text: {
+          placeholder: 'Application name',
+        },
+      },
+    },
+    {
+      type: 'text',
+      id: 'live-endpoint',
+      name: 'Live Endpoint',
+      description: 'endpoint to call when content goes live',
+    },
+    {
+      type: 'text',
+      id: 'updated-endpoint',
+      name: 'Updated Endpoint',
+      description: 'endpoint to call when live content gets updated',
+    },
+    {
+      type: 'text',
+      id: 'sunset-endpoint',
+      name: 'Sunset Endpoint',
+      description: 'endpoint to call when live content gets sunsetted',
+    },
+    {
+      type: 'text',
```

Comment:
Users don't get to edit this; why is it part of the form? This should really on't be displayed in the `view`.